docs/Makefile

```
@@ -56,7 +56,7 @@ cleannotebooks:
 
 # -W: exit on warning
 defaultdoc:
-	$(SPHINXBUILD) -b html -nW $(ALLSPHINXOPTS) $(BUILDDIR)/html
+	$(SPHINXBUILD) -b html -n $(ALLSPHINXOPTS) $(BUILDDIR)/html
```

This disables `fail_on_warning`, though in the `.readthedocs.yml` file it's still set to true?

---

True,  but the sphinx-build on RTD also has the `--keep-going` which overwrites the `-W`. A bit weird to me, but in the end it is the same
https://readthedocs.org/projects/aiida-tutorials/builds/25575956/
I just use the same flags as on RTD
```suggestion
	$(SPHINXBUILD) -b html -T -W --keep-going $(ALLSPHINXOPTS) $(BUILDDIR)/html
```

---

Yeah, a bit weird, I agree.